style.css

```diff
@@ -0,0 +1,50 @@
+eader{
+
+  background-color: salmon;
+  height: 80px;
+  font-family: cursive;
+  
+}
+
+.section-recept__title{
+  font-stretch: ultra-condensed;
+  background: #e8e8e8;
+  text-align: center;
+  font-style: oblique;
+}
+.header__title{
+ font-family: monospace;
+ background-image: linear-gradient(white, gray);
+}
+
+.header__btn {
+  background: #e6e3df;
+  text-align: center;
+  height: 50px;
+  width: 100px;
+  display: inline-block;
+}
+
+.section-recept__description{
+text-align: start;
+word-break: normal; 
+background-color: purple;
+overflow-wrap: normal;
+}
+
+.recept-list{
+text-transform: uppercase;
+background-color: green;
+border-color: #cacd58;
+border-width: 5px;
+}
+.recept-list__description{
+text-align: center;
+}
+.footer__title{ 
+font-weight: 900;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/podklu4eniestilei/pullRequest?prid=5556267)